apps/api/src/app/workflows/dto/create-notification-template.request.dto.ts

```diff
@@ -0,0 +1,65 @@
+import { IsArray, IsBoolean, IsDefined, IsOptional, IsString, MaxLength, ValidateNested } from 'class-validator';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { ICreateNotificationTemplateDto, IPreferenceChannels } from '@novu/shared';
+
+import { PreferenceChannels } from '../../shared/dtos/preference-channels';
+import { NotificationStep } from '../../shared/dtos/notification-step';
+
+export class CreateWorkflowRequestDto implements ICreateNotificationTemplateDto {
+  @ApiProperty()
+  @IsString()
+  @IsDefined()
+  name: string;
+
+  @ApiProperty()
+  @IsString()
+  @IsDefined({
+    message: 'Notification group must be provided ',
```

Comment:
```suggestion
    message: 'Workflow group must be provided ',
```
We are also changing notification group => workflow group

Reply:
@ainouzgali Should we also create a new controller for it with usecases renamed to the new one?

Reply:
I think we should, so we have all outside refactor together. But maybe in a separate pr would feel more tidy